core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -598,6 +531,21 @@ class KafkaApisTest extends Logging {
     )
   }
 
+  private def testNewBodyForwardableApi(request: RequestChannel.Request
+                                       ): Unit = {
+    val forwardCallback: ArgumentCaptor[Option[AbstractResponse] => Unit] = ArgumentCaptor.forClass(classOf[Option[AbstractResponse] => Unit])
+    val newBody: ArgumentCaptor[AbstractRequest] = ArgumentCaptor.forClass(classOf[AbstractRequest])
+
+    verify(forwardingManager).forwardRequest(
```

Comment:
all we want to verify is the method `forwardRequest` gets executed only once, right? the following assert is unnecessary as they never be null.